### PR TITLE
Update UI Canvas properties [Resolves #1448]

### LINKED ISSUE #1448: [ISSUE][DOCS] Configuring Canvas Properties documentation page multiple issues
## Describe the issue
There are multiple issues with the Configuring Canvas Properties documentation page, they will be listed here along with their Expected experience:
1. Configuring Canvas Properties contains a deprecated image of the UI Canvas pane:
ui-editor-canvas-properties
Current UI Canvas pane:
UICanvas
**Expected experience**: Documentation does not contain deprecated images.

2. Rendering Properties section - "Draw order" does not mention its default value, while other described properties do (e.g. "Is pixel aligned - Selected by default"). The default "Draw order" value is 0.
**Expected experience**: "Draw order" description mentions its default value to remain consistent with other descriptions on this documentation page.

3. Input Properties section - "Handle multi-touch" and "Handle navigation" descriptions mention that these properties are cleared by default, while they are selected by default currently in engine. "Handle multi-touch - Cleared by default. When this property is selected, ...", "Handle navigation - Cleared by default. When this property is selected, ...". 
**Expected experience**: Default value of the "Handle multi-touch" and "Handle navigation" properties is corrected in their description to remain consistent with how they appear in engine.

## Impacted page(s)

https://www.o3de.org/docs/user-guide/interactivity/user-interface/canvases/canvas-properties/

### PR DESCRIPTION
**Pull request by chanmosq**
### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md [hunk 1]
```diff
@@ -12,44 +12,43 @@ The canvas properties are displayed in the **UI Editor** **Properties** pane whe
...
 
 ## Tooltips Properties 
 
 The following property defines how a canvas displays tooltips:
-+ **Tooltip display element** - Controls which element the game displays when your users hover over an interactive element. Select an element from the drop-down list. This list is composed of the elements on your current canvas that contain the **TooltipDisplay** component. For more information about the **Tooltips** components, see [Tooltip Components](/docs/user-guide/interactivity/user-interface/components/components-tooltips).
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Tooltip display element** | Controls which element the game displays when your users hover over an interactive element. Select an element from the drop-down list. This list is composed of the elements on your current canvas that contain the **TooltipDisplay** component. For more information about the **Tooltips** components, see [Tooltip Components](/docs/user-guide/interactivity/user-interface/components/components-tooltips). | UI Elements with a TooltipDisplay component. | None |
```
Comment (micronAMZN) [inline content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md:43]: nit, the possessive adjective here struck me as odd.

```suggestion
| **Tooltip display element** | Controls which element the game displays when a user hovers over an interactive element. Select an element from the drop-down list. This list is composed of the elements on your current canvas that contain the **TooltipDisplay** component. For more information about the **Tooltips** components, see [Tooltip Components](/docs/user-guide/interactivity/user-interface/components/components-tooltips). | UI Elements with a TooltipDisplay component. | None |
```

File: content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md [hunk 2]
```diff
@@ -12,44 +12,43 @@ The canvas properties are displayed in the **UI Editor** **Properties** pane whe
...
+|-|-|-|-|
+| **Handle positional** | Causes an automatic response to positional input such as mouse movement, mouse button clicks, and touch screen input. Keyboard inputs also cause an automatic response when an interactive UI element is active (such as an elemental with a **Text Input** component on it). <br><br> Common reasons to de-select this property are if the canvas doesn't require input, or if you configure your game to handle all inputs and pass selected inputs to the UI system.| Boolean | `True` |
+| **Consume all input** | If enabled, this canvas consumes *all* input events, regardless of whether the canvas handles a specific input event. For example, if you have canvas A covering canvas B, you probably don't want canvas B handling any input while canvas A is obstructing it, so you would select this property on canvas A. Modal dialog boxes are another example of a canvas that should have this property selected. <br><br> Note that anytime a canvas is loaded, if it's set to consume all input, then it steals the inputs from any other loaded canvas. This includes canvases that are set to consume all inputs themselves. | Boolean | `False` |
```
Comment (micronAMZN) [inline content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md:30]: honestly unsure if "anytime" is correct here, but they say "any time" is always correct.

```suggestion
| **Consume all input** | If enabled, this canvas consumes *all* input events, regardless of whether the canvas handles a specific input event. For example, if you have canvas A covering canvas B, you probably don't want canvas B handling any input while canvas A is obstructing it, so you would select this property on canvas A. Modal dialog boxes are another example of a canvas that should have this property selected. <br><br> Note that any time a canvas is loaded, if it's set to consume all input, then it steals the inputs from any other loaded canvas. This includes canvases that are set to consume all inputs themselves. | Boolean | `False` |
```

**Review by micronAMZN — APPROVED** [review]
Approving with two suggestions.

File: content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md (hunk 2 above)
Comment (willihay) [reply content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md:30]: "while" would also be a good choice here.

**Review by willihay — APPROVED** [review]

**Comment by chanmosq** [comment]
@micronAMZN Applied suggestions!

**Pull request merged**